tic-tac-toe.js

```diff
@@ -1,7 +1,85 @@
 function TicTacToe() {
-
+  this.playOnPlaya = {"1": 8, "2": 1, "3":6, "4":3, "5":5, "6":7, "7":4, "8":9, "9":2};
+  this.player1 = {"1":0 , "2":0, "3":0, "4":0, "5":0, "6":0, "7":0, "8":0, "9":0};
+  this.player2 = {"1":0 , "2":0, "3":0, "4":0, "5":0, "6":0, "7":0, "8":0, "9":0};
+  this.player = this.player1;
+  this.txt = "";
+  this.headtext = "";
+  this.plays = 0;
+  this.num = 0;
+  this.turn = 0;
 }
 
 TicTacToe.prototype = {
-  
+  hasWon: function() {
+    if (this.plays > 4) {
+      if (
+      ((this.player[1] + this.player[2] + this.player[3]) == 15) ||
+      ((this.player[4] + this.player[5] + this.player[6]) == 15) ||
+      ((this.player[7] + this.player[8] + this.player[9]) == 15) ||
+
+      ((this.player[1] + this.player[5] + this.player[9]) == 15) ||
+      ((this.player[3] + this.player[5] + this.player[7]) == 15) ||
+
+      ((this.player[1] + this.player[4] + this.player[7]) == 15) ||
+      ((this.player[3] + this.player[6] + this.player[9]) == 15) ||
+      ((this.player[2] + this.player[5] + this.player[8]) == 15) )
+      {
+        return true;
+      }
+    }
+  }
 }
+
+$(document).ready( function() {
+  var ttt = new TicTacToe();
+
+  $("table tr td").on('click', function(event) {
+    ttt.txt = $(this).text();
+    ttt.num = $(this).data().user;
+    ttt.turn = $("p").text();
+
+    if (ttt.hasWon()) {
+      return false;
+    } else if ((ttt.plays%2 == 0) && (ttt.txt ==' ⚪️ ')) {
```

Comment:
why do you need to check if `ttt.plays%2 == 0`? I'm guessing this represents one player getting odd plays, and other even. It may be easier to store a variable that flips whose turn it is, like `currentTurn = 1`, so you don't need to calculate this each turn. I like the idea though!